src/debugAdapter/goDebug.ts

```diff
@@ -751,15 +751,35 @@ class GoDebugSession extends LoggingDebugSession {
 		});
 	}
 
+	private updateThreads(goroutines: DebugGoroutine[]): void {
```

Comment:
If I understand this correctly, what we are doing here is ensuring `this.threads` only has entries corresponding to existing goroutines. 

We should either rename this to something that communicates the above or add a comment above the function for the same

Reply:
To be honest `updateThreads` sounds good to me and is exactly what this routine is doing: it updates the threads list and sends the appropriate notifications to VS Code through Events interface.

Can't be more accurate than `updateAndPropagateThreadsListToVsCode`, though sounds redundant to me.